internal/isoutil/isoutil.go

```diff
@@ -1,14 +1,22 @@
 package isoutil
 
 import (
+	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 
 	diskfs "github.com/diskfs/go-diskfs"
+	"github.com/diskfs/go-diskfs/disk"
+	"github.com/diskfs/go-diskfs/filesystem"
+	"github.com/diskfs/go-diskfs/filesystem/iso9660"
 )
 
 //go:generate mockgen -package=isoutil -destination=mock_isoutil.go . Handler
 type Handler interface {
+	Extract() error
+	Create(isoPath string, size int64, volumeLabel string) error
```

Comment:
This is the confusion I was trying to avoid by not initially including `isoPath` in the handler struct.

This is the output path, which needs to be provided separately if we want to use the same handler for extracting, and then subsequently creating a new iso.

I was thinking it would be cleaner it pass the iso path to whatever function needed it as it would refer to different paths in different contexts.

What do you think @avishayt @filanov ?

Reply:
Sure, that makes sense.  One weird part in the ReadFile method is that it doesn't use the workdir, so I just put "" when initializing.

Reply:
Yeah, I suppose we should handle the case where the working directory is empty when we go to use it.
I think right now it would treat it as `/` which might be okay, but probably isn't ideal.

Reply:
Thought about it a bit more and settled on just a rename for the `Create` parameter.